graphics/libjpeg-turbo/Portfile

```diff
@@ -46,8 +47,11 @@ if {${universal_possible} && [variant_isset universal]} {
 } elseif {${configure.build_arch} in {i386 x86_64}} {
     depends_build-append port:nasm
     configure.env       ASM_NASM=${prefix}/bin/nasm
+} elseif {${configure.build_arch} eq "ppc"} {
+    if {[catch {sysctl hw.vectorunit} result] || $result == 0} {
```

Comment:
This isn't correct if binary distribution can occur (the sysctl result on the build machine could easily be different to that on the host machine), but of course we don't build binaries for ppc. Someone else could though.